test/test_geos_operations.jl

```diff
@@ -176,91 +206,145 @@ using Extents
         test_predicate(
-            GI.touches,
+            GI.touches, AG.touches,
             "POLYGON((1 1,1 5,5 5,5 1,1 1))",
             "POINT(1 1)",
             true,
         )
         test_predicate(
-            GI.crosses,
+            GI.crosses, AG.crosses,
             "POLYGON((1 1,1 5,5 5,5 1,1 1))",
             "POINT(2 2)",
             false,
         )
         test_predicate(
-            GI.within,
+            GI.within, AG.within,
             "POLYGON((1 1,1 5,5 5,5 1,1 1))",
             "POINT(2 2)",
             false,
         )
         test_predicate(
-            GI.contains,
+            GI.contains, AG.contains,
             "POLYGON((1 1,1 5,5 5,5 1,1 1))",
             "POINT(2 2)",
             true,
         )
         test_predicate(
-            GI.overlaps,
+            GI.overlaps, AG.overlaps,
             "POLYGON((1 1,1 5,5 5,5 1,1 1))",
             "POINT(2 2)",
             false,
         )
+    end
+
+    @testset "Simple methods" begin
         test_method_simple(
-            GI.union,
+            GI.union, AG.union,
             "POLYGON((1 1,1 5,5 5,5 1,1 1))",
             "POINT(2 2)",
             "POLYGON ((1 5,5 5,5 1,1 1,1 5))",
         )
         test_method_simple(
-            GI.intersection,
+            GI.intersection, AG.intersection,
             "POLYGON((1 1,1 5,5 5,5 1,1 1))",
             "POINT(2 2)",
             "POINT (2 2)",
         )
         test_method_simple(
-            GI.difference,
+            GI.difference, AG.difference,
             "POLYGON((1 1,1 5,5 5,5 1,1 1))",
             "POINT(2 2)",
             "POLYGON ((1 5,5 5,5 1,1 1,1 5))",
         )
         test_method_simple(
-            GI.symdifference,
+            GI.symdifference, AG.symdifference,
             "POLYGON((1 1,1 5,5 5,5 1,1 1))",
             "POINT(2 2)",
             "POLYGON ((1 5,5 5,5 1,1 1,1 5))",
         )
+    end
+
+    @testset "ArchGDAL methods" begin
+        @test AG.distance(
+            AG.fromWKT("POLYGON((1 1,1 5,5 5,5 1,1 1))"),
+            AG.fromWKT("POINT(2 2)"),
+        ) == 0.0
+
+        # This could be `length` to match the GeoInterface method
+        @test AG.geomlength(AG.fromWKT("LINESTRING(0 0, 10 0)")) == 10
+
+        @test AG.area(AG.fromWKT("POLYGON((0 0, 10 0, 10 10, 0 10, 0 0))")) == 100
+
+        @test GI.coordinates(
+            AG.buffer(AG.fromWKT("POLYGON((0 0, 10 0, 10 10, 0 10, 0 0))"), 1),
+        )[1][1] == [-1.0, 0.0]
+
+        @test AG.toWKT(
+            AG.convexhull(AG.fromWKT("POLYGON((0 0, 10 0, 10 10, 0 10, 0 0))")),
+        ) == "POLYGON ((0 0,0 10,10 10,10 0,0 0))"
+
+        # @test AG.asbinary(
+        #     AG.fromWKT("POLYGON((0 0, 10 0, 10 10, 0 10, 0 0))"),
+        # )[1:10] ==
+        #       UInt8[0x01, 0x03, 0x00, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 0x05]
+
+        # @test AG.astext(AG.fromWKT("POLYGON((0 0, 10 0, 10 10, 0 10, 0 0))")) ==
+        #       "POLYGON ((0 0,10 0,10 10,0 10,0 0))"
```

Comment:
Out of curiosity, why are these commented out?

Reply:
Oh because they dont exist, but maybe should to match geointerface

Reply:
Oh thanks for explaining!

Reply:
I'll add a comment and a github issue, it's basically the same as for length